### ISSUE TITLE
Input with addition class lost bootstrap class

### ISSUE THREAD

**Issue by janstieler** [formr/Formr#123]
Hi,
it looks that I can't combine a bootstrap form not with own classes.
If I add a class like this 
```php
$form = new Formr\Formr('bootstrap')
$form->text('forename', 'Vorname:', '', '', 'class="two columns"');
``` 
it lost the bootstrap classes.

Cheers

**Comment by arsalanfiroozi** [comment]
I've submitted a PR related to this issue.

**Comment by timgavin** [comment]
I've fixed this issue; testing and will push when ready.

**Comment by timgavin** [comment]
This has been fixed in v1.4. Thanks for your patience!
